assets/presto_parquet/Dockerfile

```
@@ -0,0 +1,56 @@
+    tmux \
+    unzip \
+    vim \
+    wget \
+    zip \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*;
+
+# Install Task
+RUN cd /usr/local && sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d;
+
+RUN pip3 install pyarrow;
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)